docs/reference/esql/multivalued-fields.asciidoc

```diff
@@ -17,7 +17,7 @@ POST /mv/_bulk?refresh
 
 POST /_query
 {
-  "query": "FROM mv"
+  "query": "FROM mv | LIMIT 2"
```

Comment:
@abdonpijpelink I tried to fix these tests adding the `// [TEST[warning:...` annotation, but it gets applied also to _batch requests (that don't return warnings, hence fail). Is it OK to have a LIMIT here or is there a more elegant solution?